src/videojs-contrib-hls.js

```diff
@@ -677,6 +679,8 @@ videojs.HlsSourceHandler = HlsSourceHandler;
 videojs.Hls = Hls;
 videojs.m3u8 = m3u8;
 videojs.registerComponent('Hls', Hls);
+videojs.plugin('gapSkipper', gapSkipper);
```

Comment:
it is being designed so that we can pull it out of HLS and have it stand on it's own if needed